tokens/src/themes/light/components/Image.json

```diff
@@ -2,7 +2,7 @@
   "elevation": {
     "image": {
       "thumbnail": {
-        "box-shadow": { "value": "none" }
+        "box-shadow": { "value": "0 1px 2px rgba(0, 0, 0, .075)" }
```

Comment:
[clarification] Looking at the `Image` component using the edX.org theme on the Paragon docs site for `master`, it looks like the `Image` thumbnail doesn't have a `box-shadow`?

<img width="1878" alt="image" src="https://github.com/edx/brand-edx.org/assets/2828721/3215ae7a-aaa3-488a-bd03-d86917529b87">

Reply:
Yes you are right
This pull request will be closed, and a new one with corrections will be opened [here](https://github.com/edx/brand-edx.org/pull/82)